### PR TITLE
Add MiniCPM-RobotManip LIBERO full-parameter fine-tuning

### PR DESCRIPTION
**Pull request by Yjonben**
## Description

Contributed by members of the MiniCPM-RobotManip team, this PR adds full-parameter LIBERO fine-tuning support for the released `openbmb/MiniCPM-RobotManip` checkpoint.

The example loads the released MiniCPM-RobotManip VLM and pretrained 80-D GR00T action head through its shipped `trust_remote_code` implementation, then trains both components inside starVLA.

## Motivation

MiniCPM-RobotManip is already publicly released, but starVLA does not currently provide a training path for its unified 80-D action representation. This PR adds a compact LIBERO example with the correct EE6D mapping, horizon handling, image resolution, dtype, masking, and loss construction.

## Changes

- Add the `MiniCPMRobotManip` VLM4A framework, including FP32 action-head training loss and released-checkpoint loading.
- Add a LIBERO full-parameter training example, data registry, modality metadata installer, DeepSpeed config, launcher, and local HF checkpoint exporter.
- Add configurable state/action packing dtype and image size to the shared LeRobot loader, plus an opt-in incomplete-action-chunk filter. The defaults preserve existing behavior.
- Add focused tests for EE6D-to-80D mapping, loss masking, FP32/448px data packing, and complete action horizons.

The converted EE6D dataset is publicly available at https://huggingface.co/datasets/openbmb/MiniCPM-RobotManip-LIBERO.

## Testing

- [x] 8-GPU distributed training completed through step 1,500
- [x] Benchmark evaluation results for this fine-tuning recipe

| Benchmark | Metric | Result |
|-----------|--------|--------|
| LIBERO-10 | Success rate | 93.4 |
| LIBERO-Goal | Success rate | 98.8 |
| LIBERO-Object | Success rate | 99.6 |
| LIBERO-Spatial | Success rate | 94.6 |
| LIBERO overall | Mean success rate | 96.60 |

- **Checkpoint**: https://huggingface.co/openbmb/MiniCPM-RobotManip
- **Config**: `examples/modelExtensions/MiniCPM-RobotManip/train_files/minicpm_robotmanip_libero.yaml`
- **Reproduce**: `bash examples/modelExtensions/MiniCPM-RobotManip/train_files/run_libero_train.sh`

Focused validation:

```text
python -m unittest discover -s tests -p "test_minicpm_robotmanip.py" -v
# 4 tests passed

ruff check starVLA/model/framework/VLM4A/MiniCPMRobotManip.py tests/test_minicpm_robotmanip.py examples/modelExtensions/MiniCPM-RobotManip/train_files/data_registry/data_config.py examples/modelExtensions/MiniCPM-RobotManip/train_files/export_checkpoint.py examples/modelExtensions/MiniCPM-RobotManip/train_files/install_modality.py
# All checks passed
```

## Type of Change

- [ ] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [x] New framework / benchmark integration
- [ ] Breaking change
- [ ] Documentation only
- [ ] Refactor (no behavior change)

## Checklist

- [x] No unrelated changes mixed in
- [x] New features have example config in `examples/`
- [x] No secrets, API keys, or large binaries committed
- [x] Files stay isolated — no unnecessary modification to shared modules
- [x] Shared dataloader modification is limited to opt-in/configurable behavior required for precise long-horizon samples; existing defaults are preserved and focused tests are included
- [x] Public checkpoint: https://huggingface.co/openbmb/MiniCPM-RobotManip

## Notes

The reported 96.60 result is produced by this compact, single-dataset LIBERO recipe. The released generalist model uses a broader multi-embodiment post-training setup that is outside the scope of this upstream example; no improvement over the released model is claimed.

### CODE REVIEW

**Review by JinhuiYE — APPROVED** [review]
Thanks for the contribution! I reviewed the shared dataloader changes: the new behavior is opt-in, the defaults preserve the existing 224px/FP16/unfiltered behavior, and the complete-action-chunk bounds are consistent with the configured action offsets.
